py/core/providers/email/sendgrid.py

```diff
@@ -28,6 +27,7 @@ def __init__(self, config: EmailConfig):
         self.verify_email_template_id = config.verify_email_template_id
         self.reset_password_template_id = config.reset_password_template_id
         self.client = SendGridAPIClient(api_key=self.api_key)
+        self.email_name = config.email_name
```

Comment:
I'm going to change this to `sender_name` just to make it a bit more explicit. I could see potentially wanting to have multiple email templates stored in R2R in the future, for which we might use something like an `email_name` variable.